interpreter/README.md

```diff
@@ -394,7 +399,8 @@ assertion:
   ( assert_trap <module> <failure> )         ;; assert module traps on instantiation
 
 result:
-  ( <val_type>.const <numpat> )
+  ( <val_type>.const <numpat> )              ;; numeric result
+  ( either result+ )                         ;; non-deterministic result
```

Comment:
This seems to have an unfortunate interaction with multi-value - because the either combinator is per-value, I can't express something like "either [0,0], or [1,42]". Instead I can only express [(either 0 or 1),(either 0 or 42)] which is less precise.

Reply:
Yeah, I considered lifting it, but there is no obvious syntax for writing the list of lists. In the end I decided that it's probably not worth the effort, since you could always formulate tests such that they encode the cases in a single value. We were doing that routinely before multi values.

Reply:
make sense